### PR TITLE
[PM-10632] Setup complete screen for new onboarding

### PR DESCRIPTION
**Pull request by dseverns-livefront**
## 🎟️ Tracking
https://bitwarden.atlassian.net/browse/PM-10632

## 📔 Objective
- Draw the setup complete screen when interacted on update user's onboarding status to complete

## 📸 Screenshots
image

## ⏰ Reminders before review

- Contributor guidelines followed
- All formatters and local linters executed and passed
- Written new unit and / or integration tests where applicable
- Used internationalization (i18n) for all UI strings
- CI builds passed
- Communicated to DevOps any deployment requirements
- Updated any necessary documentation or informed the documentation team

## 🦮 Reviewer guidelines

- 👍 (`:+1:`) or similar for great changes
- 📝 (`:memo:`) or ℹ️ (`:information_source:`) for notes or general info
- ❓ (`:question:`) for questions
- 🤔 (`:thinking:`) or 💭 (`:thought_balloon:`) for more open inquiry that's not quite a confirmed
  issue and could potentially benefit from discussion
- 🎨 (`:art:`) for suggestions / improvements
- ❌ (`:x:`) or ⚠️ (`:warning:`) for more significant problems or concerns needing attention
- 🌱 (`:seedling:`) or ♻️ (`:recycle:`) for future improvements or indications of technical debt
- ⛏ (`:pick:`) for minor or nitpick changes

### CODE REVIEW

File: app/src/main/java/com/x8bit/bitwarden/ui/auth/feature/accountsetup/SetupCompleteScreen.kt [hunk 1]
```diff
@@ -0,0 +1,134 @@
...
+        )
+    }
+}
+
+@Preview(showBackground = true)
+@Preview(showBackground = true, uiMode = android.content.res.Configuration.UI_MODE_NIGHT_YES)
+@Composable
+private fun SetupCompleteContent_preview() {
+    BitwardenTheme {
+        Surface {
```
Comment (dseverns-livefront) [inline app/src/main/java/com/x8bit/bitwarden/ui/auth/feature/accountsetup/SetupCompleteScreen.kt:127]: So we can preview the night/darkmode theme with dark bg.

Comment (david-livefront) [reply app/src/main/java/com/x8bit/bitwarden/ui/auth/feature/accountsetup/SetupCompleteScreen.kt:127]: Can you just add this to the preview annotation?
```showBackground = true```

Comment (dseverns-livefront) [reply app/src/main/java/com/x8bit/bitwarden/ui/auth/feature/accountsetup/SetupCompleteScreen.kt:127]: w/o the surface the background will be white like this (justMaterialThings):
image

Comment (david-livefront) [reply app/src/main/java/com/x8bit/bitwarden/ui/auth/feature/accountsetup/SetupCompleteScreen.kt:127]: Gotcha!

File: app/src/test/java/com/x8bit/bitwarden/ui/auth/feature/accountsetup/SetupCompleteViewModelTest.kt [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+            viewModel.stateFlow.value,
+        )
+    }
+
+    @Test
+    fun `When CompleteSetup action is sent user state is updated with onboarding COMPLETE`() {
+        val viewModel = createViewModel()
+        assertEquals(
+            DEFAULT_USER_ID,
+            viewModel.stateFlow.value.userId,
```
Comment (david-livefront) [inline app/src/test/java/com/x8bit/bitwarden/ui/auth/feature/accountsetup/SetupCompleteViewModelTest.kt:47]: Can we just compare it to the state?

```kotlin
assertEquals(
    SetupCompleteState(userId = DEFAULT_USER_ID),
    viewModel.stateFlow.value,
)
```

**Review by david-livefront — APPROVED** [review]

**Pull request merged**